src/backend/TrafficCourtsApi/citizen-api/Features/Lookups/LookupsService.cs

```diff
@@ -48,5 +54,10 @@ public Statute GetCountStatute(string countCode)
             if (_lookups == null) return null;
             return _lookups.statutes.FirstOrDefault(m=>m.code==Int32.Parse(countCode));
```

Comment:
If we are going to look up by id repeatedly, we could store the statutes in a Dictionary<int,Statute>.  This assumes the statute code is unique.  the above code is performing a sequential linear search over and over. How many statutes do we expect,  This is O(n) instead of O(1) with a dictionary.